crates/core/tedge_agent/src/operation_workflows/actor.rs

```diff
@@ -349,8 +374,6 @@ impl WorkflowActor {
                             ))
                             .await;
                     }
-                } else {
-                    log_file.log_info("=> sub-operation not yet launched").await;
```

Comment:
This log entry has been removed, because confusing when the sub-operation is actually a builtin operation.